extract_endpoint/src/extract_endpoint/endpoint.py

```diff
@@ -2,11 +2,11 @@
 import base64
 import typing
 from http import HTTPStatus
+import zipfile
 import flask
 from werkzeug import utils
 from azure.common import AzureMissingResourceHttpError
-from extract_endpoint import azure_utils
-from extract_endpoint import crypt_utils
+from extract_endpoint import azure_utils, crypt_utils
```

Comment:
I don't particularly mind either way, but I know that Jason prefers imports like this be on 2 separate lines.

Reply:
yeah, I just realized that while mucking around in etl.